packages/react/use-size/src/useSize.tsx

```diff
@@ -2,15 +2,11 @@
 
 import * as React from 'react';
 
-function useSize(
-  /** A reference to the element whose size to observe */
-  refToObserve: React.RefObject<HTMLElement | SVGElement>
-) {
+function useSize<T extends HTMLElement | SVGElement>(elementToObserve: T | null) {
```

Comment:
I reckon we should simplify and call the param `element`, it's self-explanatory enough.

Reply:
I agree... yay